vsts/pipelines/nightly.yml

```diff
@@ -23,160 +22,14 @@ jobs:
       echo "##vso[task.setvariable variable=TestRuntimeImages;]false"
       echo "##vso[task.setvariable variable=PushBuildImages;]true"
       echo "##vso[task.setvariable variable=PushRuntimeImages;]false"
-      echo "##vso[task.setvariable variable=PushToDockerHub;]false"
-      echo "##vso[task.setvariable variable=EmbedBuildContextInImages;]true"
-    displayName: 'Set variables'
-  - template: _buildParallel.yml
-
-- job: Job_RuntimeImages
-  displayName: Build and Test Runtime Images
-  condition: succeeded()
-  pool:
-    name: OryxLinux
-  # Building runtime images can take a long time due our PHP images
-  timeoutInMinutes: 150
-  steps:
-  - script: |
-      echo "##vso[task.setvariable variable=BuildBuildImages;]false"
-      echo "##vso[task.setvariable variable=TestBuildImages;]false"
-      echo "##vso[task.setvariable variable=BuildRuntimeImages;]true"
-      echo "##vso[task.setvariable variable=TestRuntimeImages;]true"
-      echo "##vso[task.setvariable variable=PushRuntimeImages;]true"
-      echo "##vso[task.setvariable variable=PushBuildImages;]false"
-      echo "##vso[task.setvariable variable=PushToDockerHub;]false"
       echo "##vso[task.setvariable variable=EmbedBuildContextInImages;]true"
     displayName: 'Set variables'
   - template: _buildParallel.yml
+    parameters:
+      pushToDockerHub: 'false'
 
-- job: Job_PythonIntegrationTests
```

Comment:
So... the nightly builds won't be parallel anymore? Where did these changes go?

Reply:
its still parallel